### DISCUSSION TITLE
Concourse resource for google chat spaces

### DISCUSSION THREAD

**Discussion by sethiyash** [concourse/concourse#8957] [Help & Support]
Looking for a concourse resource or some other way to notify about the pipeline status on the google chat spaces.
I have found [this](https://github.com/CloudInn/concourse-hangouts-resource) but doesn't looks like it gets managed anymore.
Other, is [this](https://github.com/spring-io/google-chat-notify-resource) one but looks like this has not been publised yet.
Is there any way if I have the Google Chat webhook URL I can post the message.

I have also tried doing:

curl -H 'Content-Type: application/json' -d "{\"text\": \"$text_content\"}" "$webhook_url"
This works from my local but when trying to do it from the concourse pipeline it throws following error:

{
  "error": {
    "code": 403,
    "message": "Method doesn't allow unregistered callers (callers without established identity). Please use API Key or other form of API consumer identity to call this API.",
    "status": "PERMISSION_DENIED"
  }
}

**Comment by marco-m-pix4d** [comment]
We wrote https://github.com/pix4d/cogito, that can be used to, independently or together:
1. send CI checks to GitHub
2. send CI messages to Google Chat
